tests/integration/relations/pgbouncer_provider/test_pgbouncer_provider.py

```diff
@@ -55,7 +55,7 @@ async def test_database_relation_with_charm_libraries(ops_test: OpsTest, applica
         ops_test.model.deploy(
             application_charm,
             application_name=CLIENT_APP_NAME,
-            resources={"application-image": "ubuntu:latest"},
```

Comment:
No resources needed anymore.

Reply:
Right. It has escaped on cherry-picking conflicts resolution. Tnx!
Cleaned both places.